spec/models/form/sales/subsections/setup_spec.rb

```diff
@@ -5,7 +5,8 @@
 
   let(:subsection_id) { nil }
   let(:subsection_definition) { nil }
-  let(:section) { instance_double(Form::Sales::Sections::Setup) }
+  let(:section) { instance_double(Form::Sales::Sections::Setup, form: instance_double(Form, start_date:)) }
+  let(:start_date) { Time.utc(2022, 4, 1) }
```

Comment:
why do we need to do this? We should try to have as few set dates in our testing as possible as it tends to make them brittle in the future.

Reply:
I agree, however by introducing the condition of dates within specific sales questions, we now require a form for the set up which requires a date.